### PR TITLE
flexget: fix build

### LINKED ISSUE #62110: flexget: Could not find a version that satisfies the requirement zxcvbn-python (from FlexGet==2.20.22) (from versions: )
## Issue description

Tried to update my server but it refuses to build because of flexget.

```
installing                                                                                                                                                                                                                                           
/build/FlexGet-2.20.22/dist /build/FlexGet-2.20.22                                                                                                                                                                                                   
62006 blocks                                                                                                                                                                                                                                         
Processing ./FlexGet-2.20.22-py2.py3-none-any.whl                                                                                                                                                                                                    
Requirement already satisfied: plumbum in /nix/store/038fqm2pfkqnnpg0n4xqsx63x2f58ymf-python3.7-plumbum-1.6.7/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.6.7)                                                                            
Requirement already satisfied: flask-login in /nix/store/q7br28fv69w138f87qh3cp23d6alzkq8-python3.7-Flask-Login-0.4.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.4.1)                                                                    
Requirement already satisfied: sqlalchemy in /nix/store/5wr1fcd6b4dl29qa6dmj5krrzb3n1hgd-python3.7-SQLAlchemy-1.2.14/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.2.14)                                                                    
Requirement already satisfied: jinja2 in /nix/store/n1dcmv0ii513dhlnllc790vfn8i9j9lj-python3.7-Jinja2-2.10.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.10.1)                                                                            
Requirement already satisfied: beautifulsoup4 in /nix/store/4dnq4cahbnprg2gja6nqicf9cxwyyc0v-python3.7-beautifulsoup4-4.7.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (4.7.1)                                                              
Requirement already satisfied: zipp in /nix/store/2apb5lkyajs6z4h105v9dinci5h0nhf4-python3.7-zipp-0.3.3/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.3.3)                                                                                  
Requirement already satisfied: python-dateutil in /nix/store/f7scw09h769xnc9chnbviw7pxjb4z0p0-python3.7-python-dateutil-2.8.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.8.0)                                                            
Requirement already satisfied: babelfish in /nix/store/v9077b38ma9fmvibrf4gqrdkxpljzq8k-python3.7-babelfish-0.5.5/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.5.5)                                                                        
Requirement already satisfied: portend in /nix/store/bkfmvpirgaggn7h4gb1da5q17mm27riz-python3.7-portend-2.3/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.3)                                                                                
Requirement already satisfied: flask in /nix/store/g6ggi7qjn007aabga980bixkaa0xisc2-python3.7-Flask-1.0.2/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.0.2)                                                                                
Requirement already satisfied: html5lib in /nix/store/v1yg31pa0blvg514b25b29v6px8ikm7y-python3.7-html5lib-1.0.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.0.1)                                                                          
Requirement already satisfied: future in /nix/store/7mmjc2sk83h0lqd96fi6hpj01a1dnyhk-python3.7-future-0.17.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.17.1)                                                                            
Requirement already satisfied: jsonschema in /nix/store/5z3pax2qlzqyy4xbx0wn0yc3gdv1kpil-python3.7-jsonschema-2.6.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.6.0)                                                                      
Requirement already satisfied: rebulk in /nix/store/q6am78ybqvqrxd4hxx5sizqzgpzg9a30-python3.7-rebulk-1.0.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.0.0)                                                                              
Requirement already satisfied: importlib-metadata in /nix/store/0jakyb1kipi89451s5ihrcbax0s8f8wn-python3.7-importlib-metadata-0.8/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.8)                                                          
Requirement already satisfied: markupsafe in /nix/store/b2wy5p5bykcnkwz5q1w8qq4qfzr4arc7-python3.7-MarkupSafe-1.1.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.1.1)                                                                      
Requirement already satisfied: aniso8601 in /nix/store/9mddxkwljdgjszla8gcw8y041p4mzgy8-python3.7-aniso8601-6.0.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (6.0.0)                                                                        
Requirement already satisfied: flask-restplus in /nix/store/dsvzd1kl4wpb5mzv49n6b2wnykq4wjcz-python3.7-flask-restplus-0.12.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.12.1)                                                            
Requirement already satisfied: rpyc in /nix/store/4925hxjf12raqqw0drryfamwzxd043fp-python3.7-rpyc-4.0.2/lib/python3.7/site-packages (from FlexGet==2.20.22) (4.0.2)                                                                                  
Requirement already satisfied: tzlocal in /nix/store/fb8axcm7zvzswx00w0bs6lqryrqgl14g-python3.7-tzlocal-1.5.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.5.1)                                                                            
Requirement already satisfied: cheroot in /nix/store/gqw890qwlwhymc0fhzfqkbak0xf2cd38-python3.7-cheroot-6.3.3/lib/python3.7/site-packages (from FlexGet==2.20.22) (6.3.3)                                                                            
Requirement already satisfied: idna in /nix/store/n2mzl8vljdksdqybihdy9mm5v7hm19q5-python3.7-idna-2.8/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.8)                                                                                      
Requirement already satisfied: webencodings in /nix/store/c1s4ps9h4vy6b22jd9jwz6h2im6ng6k4-python3.7-webencodings-0.5.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.5.1)                                                                  
Requirement already satisfied: tempora in /nix/store/8xy0iai5m6lggh9cy9q805najc30pnlw-python3.7-tempora-1.14.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.14.1)                                                                          
Requirement already satisfied: werkzeug in /nix/store/993f9ldw2dpxw5lj5a7z5vywa6xs80qx-python3.7-Werkzeug-0.15.2/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.15.2)                                                                        
Requirement already satisfied: flask-compress in /nix/store/83nnd4rgc7pz367g8p85pgyl2ww2b8l9-python3.7-Flask-Compress-1.4.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.4.0)                                                              
Requirement already satisfied: cherrypy in /nix/store/alw46yyv7jansxr7dk387qn9zi7h2654-python3.7-cherrypy-18.1.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (18.1.1)                                                                        
Requirement already satisfied: pyrss2gen in /nix/store/yddl7vlsn5fy3ynmfb3zd8s3222bdgbv-python3.7-PyRSS2Gen-1.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.1)                                                                            
Requirement already satisfied: colorclass in /nix/store/syhf1g4sll26i70jfcyjs2qx8lsbvlyg-python3.7-colorclass-2.2.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.2.0)                                                                      
Requirement already satisfied: pyyaml in /nix/store/z0sqda5bg0l4p0851nw05h7hii0jj1kr-python3.7-PyYAML-5.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (5.1)                                                                                  
Requirement already satisfied: flask-cors in /nix/store/vr9m0356bcdlwblw4gxwzb1fm8sqbrg6-python3.7-Flask-Cors-3.0.7/lib/python3.7/site-packages (from FlexGet==2.20.22) (3.0.7)                                                                      
Requirement already satisfied: itsdangerous in /nix/store/xdq121znj8padmqby7hp8i4q40xna0lh-python3.7-itsdangerous-1.1.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.1.0)                                                                  
Requirement already satisfied: click in /nix/store/yipqg38jl18vq6cqrzi28kapm2gjmg82-python3.7-click-7.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (7.0)                                                                                    
Requirement already satisfied: pynzb in /nix/store/r39sl6r3mxfjsvqs7jsx2vhazi5w6814-python3.7-pynzb-0.1.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.1.0)                                                                                
Requirement already satisfied: pytz in /nix/store/9vs1phwnsrvm1dli78ihk8rgz508malf-python3.7-pytz-2019.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (2019.1)                                                                                
Requirement already satisfied: chardet in /nix/store/wbbwikfkc7fbida822a5z9b4xmsnwm3d-python3.7-chardet-3.0.4/lib/python3.7/site-packages (from FlexGet==2.20.22) (3.0.4)                                                                            
Requirement already satisfied: urllib3 in /nix/store/ch6pz5kfg0bd3sfyf1813cpskg7lidns-python3.7-urllib3-1.24.2/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.24.2)                                                                          
Requirement already satisfied: terminaltables in /nix/store/3wab57vfbvbq2alfg21a09as3g19qk81-python3.7-terminaltables-3.1.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (3.1.0)                                                              
Requirement already satisfied: flask-restful in /nix/store/6r09f4nin02l8ck8ww4665vnfxca5a04-python3.7-Flask-RESTful-0.3.6/lib/python3.7/site-packages (from FlexGet==2.20.22) (0.3.6)                                                                
Requirement already satisfied: certifi in /nix/store/02nzlzdw0kiici9368jp5s84cpbqxkva-python3.7-certifi-2018.11.29/lib/python3.7/site-packages (from FlexGet==2.20.22) (2018.11.29)                                                                  
Requirement already satisfied: guessit in /nix/store/pjihjgwg9jscc7yz5p1mq7k1nx6vfng6-python3.7-guessit-3.0.3/lib/python3.7/site-packages (from FlexGet==2.20.22) (3.0.3)                                                                            
Requirement already satisfied: pyparsing in /nix/store/5ydkc9jcaaxlz58dr7gvyhi3gcmafsfy-python3.7-pyparsing-2.3.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.3.1)                                                                        
Requirement already satisfied: requests in /nix/store/d0wcd9mid6067i6va19lwiv29hln6n2j-python3.7-requests-2.21.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.21.0)                                                                        
Requirement already satisfied: progressbar in /nix/store/8n9sx9krvy86yanz9x36c5zdbhq7bzs2-python3.7-progressbar-2.5/lib/python3.7/site-packages (from FlexGet==2.20.22) (2.5)                                                                        
Requirement already satisfied: six in /nix/store/dy5wi2sqnhbnlpvjr8a0z96id1mq243j-python3.7-six-1.12.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (1.12.0)                                                                                  
Requirement already satisfied: feedparser in /nix/store/cljj2y3pcgnjndwdm8abl5s0axj1k900-python3.7-feedparser-5.2.1/lib/python3.7/site-packages (from FlexGet==2.20.22) (5.2.1)                                                                      
Requirement already satisfied: apscheduler in /nix/store/bfkid5s0qiy5745i7v018kga2lymhkfr-python3.7-APScheduler-3.6.0/lib/python3.7/site-packages (from FlexGet==2.20.22) (3.6.0)                                                                    
Collecting zxcvbn-python (from FlexGet==2.20.22)                                                                                                                                                                                                     
  Could not find a version that satisfies the requirement zxcvbn-python (from FlexGet==2.20.22) (from versions: )                                                                                                                                    
cannot build derivation '/nix/store/d0rph7w6dsh7p0yrsrqmaqfhkmbg8716-system-path.drv': 1 dependencies couldn't be built                                                                                                                              
cannot build derivation '/nix/store/gwjl6bqiv1bd6pi78gmpfsm7jnagd13j-nixos-system-servi-19.09pre180832.c7bcd4277cc.drv': 1 dependencies couldn't be built
```

### Steps to reproduce

1. Use Nixos Unstable
2. Add flexget as service
3. Rebuild

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`                                                                                                                                                                                                                          
 - host os: `Linux 4.19.44, NixOS, 19.09pre180029.82435a53e9a (Loris)`                                                                                                                                                                               
 - multi-user?: `yes`                                                                                                                                                                                                                                
 - sandbox: `yes`                                                                                                                                                                                                                                    
 - version: `nix-env (Nix) 2.2.2`                                                                                                                                                                                                                    
 - channels(hyper): `""`                                                                                                                                                                                                                             
 - channels(root): `"nixos-19.09pre180832.c7bcd4277cc"`                                                                                                                                                                                              
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change
Flexget uses `zxcvbn-python` instead of `zxcvbn`. Fixes https://github.com/NixOS/nixpkgs/issues/62110

cc: @sjau 
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/zxcvbn/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+buildPythonPackage rec {
+  pname = "zxcvbn";
+  version = "4.4.28";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "151bd816817e645e9064c354b13544f85137ea3320ca3be1fb6873ea75ef7dc1";
+  };
+
+  # No tests in archive
```
Comment (worldofpeace) [inline pkgs/development/python-modules/zxcvbn/default.nix]: Why not have tests from github like before?

**Review by worldofpeace — APPROVED** [review]
Fixes build :+1:

File: pkgs/development/python-modules/zxcvbn/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  doCheck = false;
```
Comment (Ma27) [inline pkgs/development/python-modules/zxcvbn/default.nix]: Isn't it possible to use the GitHub archive here to have a running test suite?

**Comment by worldofpeace** [comment]
Lol,  it's the hive mind of python packages maintainers who really like to enable tests :smile:

**Comment by Ma27** [comment]
Ouch, seems as my browser is really trolling me today, sorry for the duplicated comment...

**Comment by worldofpeace** [comment]
> Ouch, seems as my browser is really trolling me today, sorry for the duplicated comment...

I have that same problem. Usually hard refreshing does the trick.

**Comment by sjau** [comment]
Just as question: Since there's already flexget 2.21.2, wouldn't it be better to also use that?

**Comment by marsam** [comment]
@worldofpeace, @Ma27 I've updated `zxcvbn` to use GitHub archive 
@sjau I've updated to flexget 2.21.2 :)

**Comment by sjau** [comment]
looks good to me.

**Comment by worldofpeace** [comment]
Thanks for fixing this @marsam :sparkles: 

Likewise to everyone.
(always thank yourself too)

**Pull request merged**